tests/accordion-test.ts

```diff
@@ -0,0 +1,414 @@
+import { fixture, expect, html, oneEvent } from "@open-wc/testing";
+import "../src/accordion.ts";
+
+describe("AalamAccordion", () => {
+    it("initially opens items with the acc-active class", async () => {
+        const el = await fixture(html`
+            <aalam-accordion activecls="acc-active">
+                <div class="acc-title">Title 1</div>
+                <div class="acc-body acc-active">Content 1</div>
+                <div class="acc-title">Title 2</div>
+                <div class="acc-body">Content 2</div>
+            </aalam-accordion>
+        `);
+        expect(el.querySelectorAll(".acc-body.acc-active").length).to.equal(1);
+        expect(el.querySelector(".acc-body.acc-active")).to.exist;
```

Comment:
Both these condition checks imply the same thing. We are not sure at this point, if this accordion is really open or not. You could probably check if the default accordion is open by checking their display style to be block, and rest others's display style is none.